sdk/storage/storage-blob/src/sas/AccountSASPermissions.ts

```diff
@@ -68,6 +70,53 @@ export class AccountSASPermissions {
     return accountSASPermissions;
   }
 
+  /**
+   * Creates a {@link AccountSASPermissions} from a raw object which contains same keys as it
+   * and boolean values for them.
+   *
+   * @static
+   * @param {SASPermissionsLike} permissionLike
+   * @returns {AccountSASPermissions}
+   * @memberof AccountSASPermissions
+   */
+  public static from(permissionLike: SASPermissionsLike): AccountSASPermissions {
```

Comment:
>SASPermissionsLike [](start = 37, length = 18)

What if SASPermissionsLike contains permission not supported by specific kind SAS. Will you wish to report the unaligned permission?

Reply:
Won't check. A more neat way would be to have separate SASPermissionLike interfaces for each SASPermission type. But would it be necessary?

Reply:
Separated SASPermissionLike interfaces for each SASPermission type,